source/helpers/reformat-log.js

```diff
@@ -0,0 +1,53 @@
+export default (logObject) => {
+  let reformattedObj = {};
+  if (logObject.status === 'success') {
+    reformattedObj = {
+      status: logObject.status,
+      message: logObject.message,
+      logs: logObject.logs,
+    };
+    if (logObject.debug) {
+      reformattedObj.logs = logObject.debugModeLogs;
+      reformattedObj.verbose = logObject.debug;
+    }
+  } else if (logObject.code.startsWith('VALIDATION_ERROR')) {
```

Comment:
* Too much noise & and way tooooo many duplicated codes:
* You have to override a key twice (`logs`)

I bet this function should only taske 10 line of codes:
1. Set `status`, `message`, `logs`, `verbose`, `code` straightforward from `logObject`, no need any `if`s. Special case: `reformattedObj.logs = logObject.debug ? logObject.debugModeLogs : logObject.logs;`
2. If the `status` is `error` and the code starts with `VALIDATION_ERROR`, then add `errors` key to the object.